packages/eds-core-react/stories/docs/Intro.stories.mdx

```diff
@@ -1,25 +1,45 @@
-  extensive documentation.
+# Equinor Design System Storybook
+
+<Typography variant="ingress"  style={{marginTop:'24px'}}>
+  This is the storybook for the <Typography link href="https://reactjs.org/">React</Typography> implementation of the Equinor Design
+  System. Our goal is to create and provided building blocks & resources for web development in Equinor.   We have a <Typography link href="https://eds.equinor.com/">storefront</Typography> with
+  extensive documentation for components, guidelines and patterns.
 </Typography>
 
-<Typography variant="h2" style={{marginTop:'32px'}}>Packages</Typography>
-<List>
-    <List.Item><Typography link href="/?path=/docs/components">Components</Typography></List.Item>
-    <List.Item><Typography link href="/?path=/docs/icons">Icons</Typography> </List.Item>
-    <List.Item><Typography link href="/?path=/docs/tokens">Tokens</Typography> </List.Item>
-</List>
+![license](https://img.shields.io/github/license/equinor/design-system)
+![github stars](https://img.shields.io/github/stars/equinor/design-system?style=social)
 
-<Typography variant="caption" style={{marginTop:'32px'}}>
-  If you want to get in touch with the team, you’ll find us in the Equinor slack channel  <Typography link href="https://equinor.slack.com/archives/CJT20H1B9">#eds-design-system</Typography>
-</Typography>
+<Stack direction="column" align="left">
+<InfoCard title="Icons" description="Explore icons in EDS library" url="?path=/story/icons--preview" urlTitle="Browse icons"/>
+</Stack>
+
+
+
+
+<Typography variant="h2" style={{margin:"24px 0"}}>Packages</Typography>
+
+
+[![core](https://img.shields.io/npm/v/@equinor/eds-core-react?color=%23c3f3d2&label=%40equinor%2Feds-core-react&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-core-react)
 
-[![Github](https://badgen.net/badge/icon/github?icon=github&label)](https://github.com/equinor/design-system)
 
+
+[![lab](https://img.shields.io/npm/v/@equinor/eds-lab-react?color=%23c3f3d2&label=%40equinor%2Feds-lab-react&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-lab-react)
+
+
+[![tokens](https://img.shields.io/npm/v/@equinor/eds-tokens?color=%23c3f3d2&label=%40equinor%2Feds-tokens&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-tokens)
+
+[![icons](https://img.shields.io/npm/v/@equinor/eds-icons?color=%23c3f3d2&label=%40equinor%2Feds-icons&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-icons)
+
+[![utils](https://img.shields.io/npm/v/@equinor/eds-utils?color=%23c3f3d2&label=%40equinor%2Feds-utils&logo=npm&)](https://www.npmjs.com/package/@equinor/eds-utils)
+
+
+<Typography variant="h2" style={{marginTop:"32px"}}>Contact us</Typography>
+<Typography variant="caption" >
+  If you want to get in touch with the team, you’ll find us in the Equinor slack channel  <Typography link href="https://equinor.slack.com/archives/CJT20H1B9">#eds-design-system</Typography>
+</Typography>
```

Comment:
`Get in tuch with us...` The text could be more encouraging, engaging for the contact - of course if we want that 🤓

Reply:
Changed wording and fixed typo